.babelrc

```diff
@@ -1,3 +1,11 @@
 {
-    "presets": ["@babel/preset-env", "@babel/preset-react"]
-}
+    "presets": [
+        "@babel/preset-env",
+        "@babel/preset-react"
+    ],
+    "plugins": [
+        [
+            "@babel/plugin-proposal-class-properties"
+        ]
+    ]
+}
```

Comment:
Hello @Amoswachira and @Bakley ,
Please act on the alert raised by GitHub on `No newline at end of file`

Reply:
@Ogutu-Brian  I will fix that